dependencies.xml

```diff
@@ -68,7 +68,7 @@ Note all install methods after "main" take
     <dask source="pip" pip_extra="[complete]"/>
     <ray source="pip" pip_extra="[default]">2.6</ray>
     <!-- redis is needed by ray, but on windows, this seems to need to be explicitly stated -->
-    <redis source="pip" os='windows'/>
+    <redis source="pip" os='windows'>5.1.0</redis>
```

Comment:
is there a reason behind the choice of this particular version?

Reply:
yeah...trying out the latest version of redis that seemed to work . Now I tried another one